api/optimeet/groups/models.py

```diff
@@ -1,12 +1,21 @@
+from typing import Any
 from django.db import models
+from random import randint
+import django
+
+def rnd_id(): return randint(1000000000,9999999999)
```

Comment:
Could you move this function to `serializers.py` since that's the only place it's used?